pkgs/applications/video/pyca/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub, pycurl, dateutil, configobj, sqlalchemy, sdnotify, flask }:
+
+buildPythonApplication rec {
+  pname = "pyCA";
+  version = "2.1";
```

Comment:
The whitespace here isn't consistent with `sdnotify`.

Reply:
I've adapted the whitespaces.